ui/App/ProjectScreen/RemoteHelperHint.svelte

```diff
@@ -55,7 +55,7 @@
     </ExternalLink>
   </p>
   <Hoverable bind:hovering={hover}>
-    <Copyable name="shell configuration">
+    <Copyable name="shell configuration" tooltipStyle="width: fit-content;">
```

Comment:
Is the style applicable to all tooltips? Should it be moved into the `Copyable` component.

Reply:
I think it's 50/50. In some cases we want it to be `fit-content` in others we want it to take the full width. Here I moved this style into the `<Tooltip>` component and I had to make the overrides to `width: 100%` in other places: https://github.com/radicle-dev/radicle-upstream/commit/5420fdf79c298afc3c9365251cfeb4f45198b9e1.

I'm fine with either way, you can pick which one you prefer.

Reply:
Let’s leave it like it is then. I was just curious.